.circleci/config.yml

```diff
@@ -211,6 +216,20 @@ jobs:
             export KNAPSACK_PRO_TEST_DIR=turnip
             export KNAPSACK_PRO_TEST_FILE_PATTERN="turnip/**/*.feature"
             bundle exec rake "knapsack_pro:queue:rspec[-r turnip/rspec]"
+      - run:
```

Comment:
You are adding Minitest steps to the jobs that are supposed to be testing the matrix of Ruby & RSpec versions.

I think we could make it cleaner. You could add new jobs to the workflow for testing the matrix of Ruby & Minitest versions. Thanks to that:
* we have the option to test different Minitest versions in the future. 
* we would avoid testing the matrix of different Minitest & RSpec versions because it does not make sense. The testing frameworks are independent.

You could create a new story for that to not hold off on the release of this fix.


```yml
workflows:
  tests:
    jobs:
      - unit
      - integration-regular-rspec:
          name: integration-regular__ruby-<< matrix.ruby >>__rspec-<< matrix.rspec >>
          matrix:
            parameters:
              ruby: ["3.0", "3.1", "3.2"]
              rspec: ["3.10.2", "3.11.0", "3.12.2"]
      - integration-queue-rspec:
          name: integration-queue__ruby-<< matrix.ruby >>__rspec-<< matrix.rspec >>
          matrix:
            parameters:
              ruby: ["3.0", "3.1", "3.2"]
              rspec: ["3.10.2", "3.11.0", "3.12.2"]
      - integration-regular-minitest:
          name: integration-regular__ruby-<< matrix.ruby >>__minitest-<< matrix.minitest >>
          matrix:
            parameters:
              ruby: ["3.0", "3.1", "3.2"]
              minitest: ["x.x.x", ...]
      - integration-queue-minitest:
          name: integration-queue__ruby-<< matrix.ruby >>__minitest-<< matrix.minitest >>
          matrix:
            parameters:
              ruby: ["3.0", "3.1", "3.2"]
              minitest: ["x.x.x", ...]
```

Reply:
✅